run-tests.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+docker compose up -d
+docker compose exec php composer install --ignore-platform-reqs
```

Comment:
Is the `--ignore-platform-reqs` really necessary? `pimcore/pimcore:PHP8.1-cli` should have everything we need, right?

Reply:
Not sure why we need messenger here..
```
Problem 1
    - symfony/amqp-messenger is locked to version v6.0.15 and an update of this package was not requested.
    - symfony/amqp-messenger v6.0.15 requires ext-amqp * -> it is missing from your system. Install or enable PHP's amqp extension.
  Problem 2
    - symfony/amqp-messenger v6.0.15 requires ext-amqp * -> it is missing from your system. Install or enable PHP's amqp extension.
    - symfony/messenger v5.4.14 requires symfony/amqp-messenger ^5.1|^6.0 -> satisfiable by symfony/amqp-messenger[v6.0.15].
    - symfony/messenger is locked to version v5.4.14 and an update of this package was not requested.
```

Reply:
What does `composer why symfony/amqp-messenger` say?

Reply:
Ooh, okay, v5 of `symfony/messenger` requires it (v6 doesn't anymore). I didn't expect that.

Reply:
We can tackle that later...

Reply:
I leave it for now as it is.